<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depgraphV::ScanModePage</class>
 <widget class="QWidget" name="depgraphV::ScanModePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="scanModeGroupBox">
     <property name="title">
      <string>Scan Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="selectRootFoldersRadio">
        <property name="text">
         <string>By selecting root folders</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="recursiveScanCheckBox">
           <property name="text">
            <string>Scan recursively</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hiddenFoldersCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Include hidden folders</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectFilesRadio">
        <property name="text">
         <string>By selecting files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifySelectionButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Modify selection...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current selection: 0 item(s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>recursiveScanCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hiddenFoldersCheckbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectRootFoldersRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_updateEnabledFlags()</slot>
 </slots>
</ui>
